mmselfsup/datasets/pipelines/transforms.py

```diff
@@ -61,6 +86,224 @@ def __call__(self, img: torch.Tensor) -> Tuple[torch.Tensor, torch.Tensor]:
         return img, mask
 
 
+@PIPELINES.register_module()
+class BEiTMaskGenerator:
+    """Generate mask for image.
```

Comment:
Better to add some docstrings to explain the differences between `BlockwiseMaskGenerator` and `BEiTMaskGenerator`.

Reply:
Also, better to rename `BlockwiseMaskGenerator` to `SimMIMMaskGenerator`. Current name is a bit confusing.